### DISCUSSION TITLE
Subscribe to mobile push notifications when following a bill/person/jurisdiction

### DISCUSSION THREAD

**Discussion by rogerscg** [openstates/enhancement-proposals#3] [Ideas]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
The only way to follow the legislative process is by continually visiting openstates.org or by following an individual bill (via email updates). It would be great to allow users to subscribe to notifications regarding bills, officials, and jurisdictions.

https://developers.google.com/web/fundamentals/push-notifications

**Describe alternatives you've considered**
Developing native apps for Android/iOS to allow users to subscribe to notifications. This is a time-consuming process, and takes a lot of energy in the long-run to maintain. If notifications are popular on the website, then it might be worthwhile to invest more in native apps.

**Additional context**
N/A

**Comment by jamesturk** [comment]
going to move these kinds of proposals to discussions, if/when we have an action plan we'll use a ticket to track that
